browser/lib/util/defaults.js

```diff
@@ -15,11 +15,22 @@ var Defaults = {
 	transports:               ['web_socket', 'xhr', 'iframe', 'jsonp']
 };
 
+/* If an environment option is provided, the environment is prefixed to the domain
+   i.e. rest.ably.io with environment sandbox becomes sandbox-rest.ably.io */
+Defaults.environmentHost = function(environment, host) {
+	if (!environment || (String(environment).toLowerCase() === 'production')) {
```

Comment:
Well if someone pass `{ environment: true }` I would prefer to have a warning than having the code working just fine.